website/docs/r/policy_assignment.html.markdown

```diff
@@ -90,6 +97,10 @@ The following arguments are supported:
 
 * `display_name` - (Optional) A friendly display name to use for this Policy Assignment. Changing this forces a new resource to be created.
 
+* `metadata` - (Optional) The metadata for the policy assignment. This
+    is a json object representing additional metadata that should be stored
+    with the policy assignment.
```

Comment:
Could we remove these extra line breaks?

Reply:
sure. it's done. I basically copied this section from the policy definition page.